node/src/BaseClient.ts

```diff
@@ -5165,6 +5166,36 @@ export class BaseClient {
         preferReplica: connection_request.ReadFrom.PreferReplica,
     };
 
+    /**
+     * Sets the last delivered ID for a consumer group.
+     *
+     * See https://valkey.io/commands/xgroup-setid for more details.
+     *
+     * since Valkey 7.0 and above
```

Comment:
According to docs, only `entries_read` is since Valkey 7.0. Should also be `@remarks Since...`